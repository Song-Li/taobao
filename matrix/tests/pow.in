SET c = sequence 1 1
SHOW c
SET e = matrix.pow c 1
SHOW e
SET f = matrix.pow c 2
SHOW f
SET h = matrix.pow c 3
SHOW h
