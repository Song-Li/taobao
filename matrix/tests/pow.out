> ok

> 1.00 2.00
3.00 4.00

> ok

> 1.00 2.00
3.00 4.00

> ok

> 7.00 10.00
15.00 22.00

> ok

> 37.00 54.00
81.00 118.00

> 
bye
